photonlib-java-examples/simaimandrange/src/main/java/frc/robot/Robot.java

```diff
@@ -24,13 +24,16 @@
 
 package frc.robot;
 
+import static frc.robot.Constants.*;
```

Comment:
I can see why it could be confusing to new users, but it also cleans up a ton of `Constants.` everywhere. I'm fine with removing the static if you want

Reply:
If it's more than like 5 things (which I think it is?) I think it's fine